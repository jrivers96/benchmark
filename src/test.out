att,read_bytes,total_seconds,bytes_per_second
'all',0,2.38393e-313,inf
