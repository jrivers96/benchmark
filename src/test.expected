count
256
count
10
count
256
att,count,bytes,chunks,min_count,avg_count,max_count,min_bytes,avg_bytes,max_bytes
'all',10000000,170002720,40,1000000,1e+06,1000000,48,4.25007e+06,9000072
att,count,bytes,chunks,min_count,avg_count,max_count,min_bytes,avg_bytes,max_bytes
'a',10000000,80000720,10,1000000,1e+06,1000000,8000072,8.00007e+06,8000072
'b',10000000,90000720,10,1000000,1e+06,1000000,9000072,9.00007e+06,9000072
'c',10000000,800,10,1000000,1e+06,1000000,80,80,80
'EmptyTag',10000000,480,10,1000000,1e+06,1000000,48,48,48
